chia/wallet/wallet_state_manager.py

```diff
@@ -655,7 +661,7 @@ async def coin_added(
         wallet_type: WalletType,
         height: uint32,
         all_outgoing_transaction_records: List[TransactionRecord],
-    ):
+    ) -> WalletCoinRecord:
```

Comment:
I don't see the call to this function that use the new return value. Is this change necessary?

Reply:
Not necessary, I started in direction of rebuilding the cache by tracking of the writes that failed first but then changed direction to just rebuild from scratch. This is artifact, I'll remove this.